content/channels/options/deltas.textile

```diff
@@ -53,82 +85,101 @@ This will cause delta messages to be generated by the server and sent to the cli
     plugins: {
-      vcdiff: vcdiffPlugin
-    }
-  });
-  const channel = realtime.channels.get('{{RANDOM_CHANNEL_NAME}}', {
+        vcdiff: vcdiffPlugin
+    },
+    log: { level: 4 } // optional
+});
+
+const channel = realtime.channels.get('your-ably-channel', {
     params: {
-      delta: 'vcdiff'
+        delta: 'vcdiff'
     }
-  });
+});
 
-  await channel.subscribe((message) => {
-    console.log('Received message: ', msg);
-  });
+channel.subscribe(msg => console.log("Received message: ", msg));
 ```
 
 ```[realtime_java]
-  AblyRealtime ably = new AblyRealtime("{{API_KEY}}")
-  Channel channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}", new ChannelOptions{{params = Map.of("delta", "vcdiff")}});
-  channel.subscribe(new MessageListener() {
-    @Override
-    public void onMessage(Message message) {
-      System.out.println("Received `" + message.name + "` message with data: " + message.data);
-    }
-  });
+AblyRealtime ably = new AblyRealtime("{{API_KEY}}")
+Channel channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}", new ChannelOptions{{params = Map.of("delta", "vcdiff")}});
+channel.subscribe(new MessageListener() {
+  @Override
+  public void onMessage(Message message) {
+    System.out.println("Received `" + message.name + "` message with data: " + message.data);
+  }
+});
 ```
 
 ```[realtime_swift]
-  let options = ARTClientOptions(key: key)
-  let client = ARTRealtime(options: options)
-  let channelOptions = ARTRealtimeChannelOptions()
-  channelOptions.params = [
-    "delta": "vcdiff"
-  ]
-
-  let channel = client.channels.get(channelName, options: channelOptions)
+let options = ARTClientOptions(key: key)
+let client = ARTRealtime(options: options)
+let channelOptions = ARTRealtimeChannelOptions()
+channelOptions.params = [
+  "delta": "vcdiff"
+]
+
+let channel = client.channels.get(channelName, options: channelOptions)
 ```
 
 ```[realtime_csharp]
-  var clientOptions = new ClientOptions();
-  clientOptions.Key = "{{API_KEY}}";
-  clientOptions.Environment = AblyEnvironment;
-  var ably = new AblyRealtime(clientOptions);
-
-  var channelParams = new ChannelParams();
-  channelParams.Add("delta", "vcdiff");
-  var channelOptions = new ChannelOptions();
-  channelOptions.Params = channelParams;
-  var channel = ably.Channels.Get("{{RANDOM_CHANNEL_NAME}}", channelOptions);
-
-  channel.Subscribe(message => {
-      Console.WriteLine(message.Data.ToString());
-  });
+var clientOptions = new ClientOptions();
+clientOptions.Key = "{{API_KEY}}";
+clientOptions.Environment = AblyEnvironment;
+var ably = new AblyRealtime(clientOptions);
+
+var channelParams = new ChannelParams();
+channelParams.Add("delta", "vcdiff");
+var channelOptions = new ChannelOptions();
+channelOptions.Params = channelParams;
+var channel = ably.Channels.Get("{{RANDOM_CHANNEL_NAME}}", channelOptions);
+
+channel.Subscribe(message => {
+    Console.WriteLine(message.Data.ToString());
+});
 ```
 
+  blang[javascript,nodejs].
```

Comment:
I'd maybe just put this under the original blang above rather than separating out.